templates/settings.py.erb

```diff
@@ -1,3 +1,16 @@
+################################################################################
+# !!! WARNING: DO NOT EDIT THIS FILE !!!                                       #
+################################################################################
+# File managed by Puppet.                                                      #
+# Module: pulpcore                                                             #
+################################################################################
+# Not only are your edits likely to be overwritten, there is also a strong     #
+# possibility of breaking your system if you change configuration here without #
+# making required changes elsewhere. Refer to the documentation you used to    #
+# install Pulpcore to determine the safe and persistent way to modify the      #
+# configuration.                                                               #
```

Comment:
My only question is if we should avoid `you` pronouns here. So `Not only are edits likely`, `possibility of breaking the system`, `if changes are made`, `Refer to the documentation used to install`

Reply:
Updated to use passive voice consistently / remove 2nd person pronouns. Thanks